tests/test_runserver_watch.py

```diff
@@ -75,6 +80,7 @@ async def test_multiple_file_change(event_loop, mocker):
     await app_task._session.close()
 
 
+@needs_py38_test
 @non_windows_test
```

Comment:
Have you tested to see if any other tests now pass on Windows?

Reply:
Yes, I was able to drop the two `@non_windows_test`s. The other skis are due to `@forked`.